Aptoide/appcoins-wallet-android
# [MOB-616] - Added payment methods icons to onboarding.

**What does this PR do?**

Added payment methods icons to onboarding.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] OnboardingActivity.kt

**How should this be manually tested?**

  Open the wallet for the first time to see the onboarding screen.
  In the third page, payment methods icons should be shown.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-616](https://aptoide.atlassian.net/browse/MOB-616)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

No


**Code Review Checklist**
